Common/Data/Custom/Tiingo/TiingoNews.cs

```diff
@@ -102,6 +102,12 @@ public class TiingoNews : IndexedBaseData
         /// <returns>The <see cref="SubscriptionDataSource"/> instance to use</returns>
         public override SubscriptionDataSource GetSourceForAnIndex(SubscriptionDataConfig config, DateTime date, string index, bool isLiveMode)
         {
+            if (isLiveMode)
```

Comment:
`GetSource` will be called first and based on it `GetSourceForAnIndex` _might_ be called (for example for backtesting -> `FileFormat.Index` ) - So I think we want this `if` there?

Reply:
Oops! the change was actually meant to be in `GetSource` :)